src/app/user/user.service.spec.ts

```diff
@@ -45,34 +45,34 @@ describe('Service: User service', () => {
   ));
 
   let testUser = {
-    "attributes": {
-      "fullName": "name",
-      "imageURL": "",
-      "username": "myUser"
+    'attributes': {
```

Comment:
Changes like this are not related and should be their own PR. If this PR is rejected then we would lose this.

Reply:
These are some linting error that i fix when i see one while working on something. That's why they are together like this. Will keep in mind to separate these next time.